src/sentry/runner/commands/cleanup.py

```diff
@@ -113,9 +118,6 @@ def is_filtered(model):
     else:
         if not silent:
             click.echo("Removing old NodeStore values")
-        if is_filtered('NodeStore'):
```

Comment:
As discussed, the only current `NodeStore` backend that supports direct deletions is the django backend, and we agree that filtering out that backend seems a little weird.